ui/app/components/ui/popover/index.scss

```diff
@@ -107,4 +110,16 @@
       margin: 0 auto;
     }
   }
+
+  &-arrow {
+    width: 22px;
+    height: 22px;
+    background: white;
+    position: absolute;
+    transform: rotate(45deg);
+    top: -6px;
+    left: 24px;
+    display: flex;
```

Comment:
Nit: this doesn't seem to do anything. The default `block` display works fine.

Reply:
addressed in 9bfe721f8